Makefile

```diff
@@ -80,11 +80,8 @@ help:
 all: manager
 
 # Run tests
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
 test: generate fmt vet manifests
-	mkdir -p ${ENVTEST_ASSETS_DIR}
```

Comment:
Why are we removing these? I thought it is essential to test Go operators?

Reply:
These are not necessarily required. These assets were actually causing test failures, so I removed them. So long as Kubebuilder is properly installed, the tests will still run. If it turns out that we do need these, then they can be put back. But in the meantime, I think they can be removed.